src/hist.jl

```diff
@@ -1,5 +1,28 @@
 
 # ---------------------------------------------------------------------------
+# StatsBase.Histogram
+
+_hist_edge_centers(edge::AbstractVector) = (edge[2:end] + edge[1:end-1]) / 2
+
+@recipe function f{T, E}(h::Histogram{T, 1, E})
+    x := _hist_edge_centers(h.edges[1])
+    y := h.weights
+    seriestype := :bar
```

Comment:
I think this should use `-->`, not `:=`.  This way someone can draw a path or something else if they want.

Instead of setting x and y and returning `()`, I'd just return the centers and weights as a tuple.  Otherwise you'll lose proper processing of the fillrange.